core/templates/pages/admin-page/services/admin-router.service.ts

```diff
@@ -29,6 +29,13 @@ export class AdminRouterService {
   currentTabHash: string = (
     AdminPageConstants.ADMIN_TAB_URLS.ACTIVITIES);
 
+  /**
+   * Iterates through the ADMIN_TAB_URLS map and returns the
+   * tab name corresponding to the hash.
+   * @param {string} tabHash Represents hash name of a tab URL.
+   * @returns {string|null} Tab name of the hash value
```

Comment:
`The corresponding tab name, or null if the URL hash does not correspond to a valid tab.`

(Also, indent the continuation line by 2.)

Reply:
Done